content/options.xul

```diff
@@ -57,7 +57,7 @@
         onclick="window.openDialog('chrome://tblatex/content/help.html', '',
           'all,chrome,dialog=no,status,toolbar,width=640,height=480');" />
     </hbox>
-    <description>Use __REPLACE_ME__ as a place marker, if you want to have it automatically highlighted.</description>
+    <description>Use __REPLACE_ME__ as a place marker for your LaTeX expression.</description>
```

Comment:
I think "placeholder" is the correct term, other than that looks good!

Reply:
I think, both terms are ok, but to be honest, "place marker" always sounded a bit strange to me (but I kept the original term - There is always room for improvement, as they say!)

When greping for "marker" I noticed, that `insert.js` also uses the placeholder to select it. I also changed it to use the new and old placeholder.